### PR TITLE
Support in-app messaging on tvOS apps

### PR DESCRIPTION
**Pull request by christibbs**
Enables custom UI display of in-app messages on tvOS.

Developers can implement the `InAppMessagingDisplay` protocol to show messages on tvOS.

Example:
```
func displayMessage(_ messageForDisplay: InAppMessagingDisplayMessage,
                      displayDelegate: InAppMessagingDisplayDelegate) {
    DispatchQueue.main.async {
      guard let modal = messageForDisplay as? InAppMessagingModalDisplay else {
        return
      }

      let alert = UIAlertController(
        title: modal.title,
        message: modal.bodyText,
        preferredStyle: .alert
      )

      let imageAlertAction = UIAlertAction(title: "", style: .default, handler: nil)
      let image = UIImage(data: (modal.imageData!.imageRawData)!)?
        .withRenderingMode(.alwaysOriginal)
      imageAlertAction.setValue(image, forKey: "image")

      alert.addAction(imageAlertAction)

      alert
        .addAction(UIAlertAction(title: modal.actionButton?.buttonText, style: .default,
                                 handler: { action in
                                   UIApplication.shared.open(modal.actionURL!)
                                 }))
      alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))

      self.present(alert, animated: true, completion: nil)
    }
  }
```

Screenshot:
Simulator Screen Shot - Apple TV 4K - 2020-12-14 at 15 56 36

### CODE REVIEW

File: FirebaseInAppMessaging/Sources/Analytics/FIRIAMAnalyticsEventLoggerImpl.m [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 
 #import <TargetConditionals.h>
-#if TARGET_OS_IOS
+#if TARGET_OS_IOS || TARGET_OS_TV
```
Comment (ryanwilson) [inline FirebaseInAppMessaging/Sources/Analytics/FIRIAMAnalyticsEventLoggerImpl.m:18]: Nit: the end of the file has `#endif  // TARGET_OS_IOS` can you also include the TV conditional too?

**Review by ryanwilson — APPROVED** [review]
Awesome! Please fix CI then it's good to go.

**Comment by chliangGoogle** [comment]
It would be great to add tvOS in workflow too for presubmit check.

File: FirebaseInAppMessaging/Sources/Analytics/FIRIAMAnalyticsEventLoggerImpl.m (hunk 1 above)
Comment (christibbs) [reply FirebaseInAppMessaging/Sources/Analytics/FIRIAMAnalyticsEventLoggerImpl.m:18]: Resolved in 1307336.

**Pull request merged**